src/Entity/Agent.php

```diff
@@ -332,14 +332,14 @@ public function setPassword(?string $password): static
         return $this;
     }
 
-    public function getRemainder(): ?float
+    public function getHolidayRemainder(): ?float
```

Comment:
You need to replace getRemainder with getHolidayRemainder in the following files : 
* src/Controller/HolidayController.php
* tests/Controller/HolidayControllerHoursTest.php
* tests/Controller/HolidayControllerDaysTest.php

You can use the following command to check : `grep --exclude-dir={.git,var,vendor} -ri "getRemainder" *`